### ISSUE TITLE
how do I get the files

### ISSUE THREAD

**Issue by quinhone** [viral-vector/vue-upload-image#4]
Hello, how do I get the files, tried onchange, but does not execute.

**Comment by viral-vector** [comment]
I need better explanation of what you are looking to do

**Comment by jEstevezRod** [comment]
You really need to improve your documentation...

**Comment by innoflash** [comment]
As in what if i do not want to upload but i want the images as is so that i can manipulate them and do what i want with them  before uploading them. 

Can you `return `the images

**Comment by viral-vector** [comment]
version 1.1.5 published with a few goodies. With this update, via @innoflash suggestion & PR. You may access the files by listening to a few events calls. 
* upload-image-loaded: after an image has been loaded to the preview area
* upload-image-clicked: after an end-user clicks on an image in the preview area
* upload-image-submit: after an end-user click the submit/upload button

**Comment by abrhambas01** [comment]
What upload button do you mean @viral-vector , does this refer to the button of the overall form that included the upload-image was included if the form includes the First name and etc.

**Comment by viral-vector** [comment]
@abrhambas01 correct. The upload button for the image uploader set by "url" & "button_html"
